### PR TITLE
Update the doc in Repo API

### PR DESCRIPTION
**Pull request by bliuredhat**
We have remove rhe release type validation in repo validation,
So update the doc in sync.

JIRA:PDC-1481

### CODE REVIEW

**Comment by ycheng-aa** [comment]
:+1:

File: pdc/apps/repository/views.py [hunk 1]
```diff
@@ -54,16 +54,6 @@ def create(self, *args, **kwargs):
         content category. If and only if the content category is `debug`, the
         name must contain `debug` substring.
```
Comment (simozhan) [inline pdc/apps/repository/views.py]: This part also needs to be removed, right?

Comment (bliuredhat) [inline pdc/apps/repository/views.py]: yes

**Comment by simozhan** [comment]
looks good to me

**Pull request merged**